docs/hypernode-platform/api.md

```diff
@@ -0,0 +1,20 @@
+---
+myst:
+  html_meta:
+    description: This table of contents gives you a summary of all Hypernode platform
+      knowledge base articles that include information about varnish.
+    title: Hypernode-API | Hypernode platform
+redirect_from:
+  - /en/hypernode/api/
```

Comment:
What is this redirect_from based on?

Reply:
I copied the myst: block from other articles, maybe redirect works only for articles that were migrated while changing from old docs? Not sure. Currently it's poitning to the folder of all api articles: 
![image](https://github.com/ByteInternet/hypernode-docs-next/assets/44441199/30c6db2a-682f-452d-a153-4030c9184622)

Reply:
If it doesn't have any special meaning I think we should just remove it :)